llvm/test/SYCLLowerIR/printf_addrspace/generic_as_negative_checks.ll

```diff
@@ -10,6 +10,10 @@
 ; RUN: opt < %S/generic_as_variadic.ll --SYCLMutatePrintfAddrspace -S | FileCheck %s --check-prefix=CHECK-BUILTIN
 ; RUN: opt < %S/generic_as_variadic.ll --SYCLMutatePrintfAddrspace -S --enable-new-pm=1 | FileCheck %s --check-prefix=CHECK-BUILTIN
 
+; generic_as_variadic_no_opt.ll
+; RUN: opt < %S/generic_as_variadic_no_opt.ll --SYCLMutatePrintfAddrspace -S | FileCheck %s --check-prefixes=CHECK-WRAPPER,CHECK-BUILTIN
+; RUN: opt < %S/generic_as_variadic_no_opt.ll --SYCLMutatePrintfAddrspace -S --enable-new-pm=1 | FileCheck %s --check-prefixes=CHECK-WRAPPER,CHECK-BUILTIN
```

Comment:
```suggestion
; RUN: opt < %S/generic_as_variadic_no_opt.ll --SYCLMutatePrintfAddrspace -S --enable-new-pm=0 | FileCheck %s --check-prefixes=CHECK-WRAPPER,CHECK-BUILTIN
; RUN: opt < %S/generic_as_variadic_no_opt.ll -passes=SYCLMutatePrintfAddrspace -S | FileCheck %s --check-prefixes=CHECK-WRAPPER,CHECK-BUILTIN
```
This does the same checks for legacy and new PMs. However, when new PM becomes default suggested options will keep checks for both PMs, while current options will check only for new PM twice.
I'd suggest to change PM selection options for all other checks as well.

Reply:
I think either option is fine. So, it's up to you.

Reply:
https://github.com/intel/llvm/pull/5302/commits/bb9b948b10d55bdc680cae13d9f55d8dd945e69d for the new checks, https://github.com/intel/llvm/pull/5302/commits/74d315364d4cc723425a0b29a301b80cfbebd8bd for the existing ones.